website/conferences/views.py

```diff
@@ -93,7 +91,15 @@ def add_poster_by_email(conference, message):
         else:
             set_password_url = None
 
-        node, node_created = utils.get_or_create_node(message.subject, user)
+        node, node_created = Node.objects.get_or_create(
+            title__iexact=message.subject,
+            is_deleted=False,
+            _contributors__guids___id=user._id,
```

Comment:
Can this just be `contributors=user` or `_contributors=user`?

Reply:
I actually think this is necessary, although I don't have a great explanation for why other than: 

`contributors=user` errors with:
```
FieldError: Invalid field name(s) for model Node: 'contributors'.
```
and `_contributors=user` errors with:
```
ValueError: "<AbstractNode: (title='testing', category='') with guid None>" 
needs to have a value for field "id" before this many-to-many relationship 
can be used.
```